2_introduction_to_rust.md

```diff
@@ -805,11 +805,17 @@ enum Expr {
 }
 
 fn main() {
-    let e = Expr::BinOp {l: Box::new(Expr::BinOp {l: Box::new(Expr::Literal(12)),
-                                                  op: Operator::Plus,
-                                                  r: Box::new(Expr::Literal(15))}),
-                         op: Operator::Plus,
-                         r: Box::new(Expr::Literal(12))};
+    let e =
```

Comment:
This will make the code example too big for the slide (check by rendering it first with `build.sh`)